## dpatch by  <marceloshima@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change the path to the keyboard configuration file

--- a/programs/Xserver/os/oscolor.c
+++ b/programs/Xserver/os/oscolor.c
@@ -53,7 +53,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
-static char* nxAltRgbPaths[] = {"/usr/NX/share/rgb", "/usr/share/X11/rgb", "/etc/X11/rgb"};
+static char* nxAltRgbPaths[] = {"/usr/share/nx/rgb", "/usr/NX/share/rgb", "/usr/share/X11/rgb", "/etc/X11/rgb"};
 static char _NXRgbPath[1024];
 
 #endif
